casadm/Makefile

```diff
@@ -77,7 +77,7 @@ CFLAGS    += -O0 -g
 else
 CFLAGS    += -O2 -D_FORTIFY_SOURCE=2
 endif
-CFLAGS    += -Wall -z relro -z now -fstack-protector -fPIC -Wformat -Wformat-security -fno-strict-aliasing
+CFLAGS    += -Werror -z relro -z now -fstack-protector -fPIC -Wformat -Wformat-security -fno-strict-aliasing
```

Comment:
I suppose -Werror should be supplementary to the -Wall flag and not instead of it.